### DISCUSSION TITLE
Placeholders for top 10 !number! of votes

### DISCUSSION THREAD

**Discussion by thboehi** [Zrips/CMI#3402] [Feature Requests]
**As we have separate placeholders for money:**
```
%cmi_baltop_money_[1-10]%
%cmi_baltop_name_[1-10]%
```
It would be great to have this for the votes, I tried with ParseOther Placeholders, but doesn't work.

```
%cmi_votetop_name_[1-10]%
%cmi_votetop_count_[1-10]%
```

**Comment by CloudeLecaw** [comment]
Didnt you know there is a topvote allready?
/cmi  votetop
Have you checkt the language file there? Im sure there is something like a placeholder allready...
%cmi_votetop_[1-10]% is the placeholder so you can define top 1 or top 5 up to top 10

So this one is not enough for you right? 
You want a splitted placeholder for name   votetop and Counts votetop right?

**Comment by thboehi** [comment]
I know all placeholders, and I know the plugin by heart, I translated it. Yes this is what I want.

**Comment by CloudeLecaw** [comment]
hmm i translated it too^^

**Comment by thboehi** [comment]
That wasn't the point... -.-

**Comment by CloudeLecaw** [comment]
You got a placeholder wirh last update. May wanne close?
